tests/lib/rules/no-side-effects.js

```diff
@@ -225,5 +231,57 @@ eslintTester.run('no-side-effects', rule, {
       output: null,
       errors: [{ message: ERROR_MESSAGE, type: 'AssignmentExpression' }],
     },
+
+    // Event functions (from this):
+    {
+      code: 'computed(function() { this.send(); })',
+      output: null,
+      errors: [{ message: ERROR_MESSAGE, type: 'CallExpression' }],
+    },
+    {
+      code: 'computed(function() { this.sendAction(); })',
+      output: null,
+      errors: [{ message: ERROR_MESSAGE, type: 'CallExpression' }],
+    },
+    {
+      code: 'computed(function() { this.sendEvent(); })',
+      output: null,
+      errors: [{ message: ERROR_MESSAGE, type: 'CallExpression' }],
+    },
+    {
+      code: 'computed(function() { this.trigger(); })',
+      output: null,
+      errors: [{ message: ERROR_MESSAGE, type: 'CallExpression' }],
+    },
+
+    // Event functions (from Ember):
+    {
+      code: 'import Ember from "ember"; computed(function() { Ember.send(); })',
+      output: null,
+      errors: [{ message: ERROR_MESSAGE, type: 'CallExpression' }],
+    },
+    {
+      code: 'import Ember from "ember"; computed(function() { Ember.sendAction(); })',
+      output: null,
+      errors: [{ message: ERROR_MESSAGE, type: 'CallExpression' }],
+    },
+    {
+      code: 'import Ember from "ember"; computed(function() { Ember.sendEvent(); })',
+      output: null,
+      errors: [{ message: ERROR_MESSAGE, type: 'CallExpression' }],
+    },
+    {
+      code: 'import Ember from "ember"; computed(function() { Ember.trigger(); })',
+      output: null,
+      errors: [{ message: ERROR_MESSAGE, type: 'CallExpression' }],
+    },
+
+    {
+      // Imported sendEvent function:
+      code:
+        'import { sendEvent } from "@ember/object/events"; computed(function() { sendEvent(); })',
```

Comment:
Could you import it under a different name, to make sure it's not assuming it's named `sendEvent`?

Reply:
Added.